### PR TITLE
vg/vgimg: add Image to vgimg.Canvas

### PR DESCRIPTION
**Pull request by sbinet**
Fixes gonum/plot#364.

### CODE REVIEW

File: vg/vgimg/vgimg.go [hunk 1]
```diff
@@ -164,6 +164,14 @@ func UseImageWithContext(img draw.Image, gc draw2d.GraphicContext) option {
 	}
 }
 
+// Image returns the image the canvas is drawing to.
+//
+// Users are not supposed to directly modify the image once the canvas
```
Comment (kortschak) [inline vg/vgimg/vgimg.go]:
```
// Users should not directly modify the returned image.
```

Are you saying that no modification is allowed? or just things like size. I think this needs to be fleshed out more.

Comment (sbinet) [inline vg/vgimg/vgimg.go]: yes, the size (ie: via `draw.Image.Bounds()` and then `image.Rectange{ Min, Max Point }`).

Comment (sbinet) [inline vg/vgimg/vgimg.go]: fixed.

Comment (kortschak) [inline vg/vgimg/vgimg.go]: OK. So then probably we should say "The dimensions of the returned image must not be modified."

Comment (sbinet) [inline vg/vgimg/vgimg.go]: done.

Comment (kortschak) [inline vg/vgimg/vgimg.go]: Thanks.

**Pull request merged**